cdc/sink/mysql/mysql.go

```diff
@@ -273,6 +286,11 @@ func (s *mysqlSink) flushRowChangedEvents(ctx context.Context) {
 		if len(resolvedTxnsMap) != 0 {
 			s.dispatchAndExecTxns(ctx, resolvedTxnsMap)
 		}
+		for _, worker := range s.workers {
```

Comment:
Could you add a comments to tell why we need to go to `outer` if a worker `!isNormal`?